Foundation/NSJSONSerialization.swift

```diff
@@ -97,7 +97,26 @@ public class NSJSONSerialization : NSObject {
     /* Generate JSON data from a Foundation object. If the object will not produce valid JSON then an exception will be thrown. Setting the NSJSONWritingPrettyPrinted option will generate JSON with whitespace designed to make the output more readable. If that option is not set, the most compact possible JSON will be generated. If an error occurs, the error parameter will be set and the return value will be nil. The resulting data is a encoded in UTF-8.
      */
     public class func dataWithJSONObject(obj: AnyObject, options opt: NSJSONWritingOptions) throws -> NSData {
-        NSUnimplemented()
+        guard obj is NSArray || obj is NSDictionary else {
```

Comment:
@parkera That's how I read it too. Now, if @ianpartridge does want to detect the native Swift containers as well, what would you recommend is the right approach? I wrote my own JSON parser [Jay](https://github.com/czechboy0/Jay), where I had to opt in for [ugly (almost) exhaustive](https://github.com/czechboy0/Jay/blob/master/Sources/Jay/NativeTypeConversions.swift#L96) checking of all Array/Dictionary types, because of the lack of automatic bridging on Linux. Is there a better way to do it that I missed?